### PR TITLE
docs: edit "Getting started" page

### PR DESCRIPTION
**Pull request by najikahalsema**
## Description
Updating the docs and editing them. This is the first in the few that we talked about including in our onboarding doc, so I made a few changes. Let me know if there are things that you would like to see updated further. 

## Motivation and Context

New hires require a fresh look at our docs :) 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: documentation/guides/getting-started.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Getting started
 
-For the fastest possible start, check out the `@spectrum-web-components/bundle` package which includes all of the elements defined by Spectrum Web Components in one easy to import entry point. Copy and paste the following HTML sample, and you can be off and running.
+For the fastest possible start, check out the `@spectrum-web-components/bundle` package, which includes all of the elements defined by Spectrum Web Components in one easy-to-import entry point.
+
+## What you're getting
+
+The `@spectrum-web-components/bundle` package is literally _everything_ that Spectrum Web Components has to offer. This is why when bundled, <sp-link target="_blank" href="https://bundlephobia.com/result?p=@spectrum-web-components/bundle">it weighs as large as it does</sp-link>, and is why we DO NOT suggest leveraging this technique in a production application. That being said, it is the shortest path to getting right into developing with Spectrum Web Components, so dive right in. Copy and paste the following HTML sample, and you’ll be up and running. Good luck, and have fun!
```
Comment (Westbrook) [inline documentation/guides/getting-started.md:7]:
```suggestion
The `@spectrum-web-components/bundle` package is literally _everything_ that Spectrum Web Components has to offer. This is why when bundled <sp-link target="_blank" href="https://bundlephobia.com/result?p=@spectrum-web-components/bundle">it weighs as large as it does</sp-link> and is why we DO NOT suggest leveraging this technique in a production application. That being said, it is the shortest path to getting right into developing with Spectrum Web Components, so dive right in. Copy and paste the following HTML sample, and you’ll be up and running. Good luck, and have fun!
```
Was this too many commas, or not enough?

File: documentation/guides/getting-started.md [hunk 2]
```diff
@@ -14,7 +18,7 @@ For the fastest possible start, check out the `@spectrum-web-components/bundle`
         onclick="alert('I was clicked');"
     >Click me!</sp-button>
     <!-- End content requiring theme application. -->
-</sp-theme>
+</sp-theme>ß
```
Comment (Westbrook) [inline documentation/guides/getting-started.md:21]: Erroneous character?

File: documentation/guides/getting-started.md [hunk 3]
```diff
@@ -23,13 +27,10 @@ The code above renders to the browser as follows (be patient while the JS is loa
...
+Visually, all Spectrum Web Components are an expression of the design tokens that are specified by Spectrum, Adobe's design system. On the web, these tokens are applied as CSS Custom Properties. Not only do these properties ensure that each element is delivered according to your application’s specifications, but for all colour/scale/content direction specifications across Spectrum.
```
Comment (Westbrook) [inline documentation/guides/getting-started.md:32]:
```suggestion
Visually, all Spectrum Web Components are an expression of the design tokens that are specified by Spectrum, Adobe's design system. On the web, these tokens are applied as CSS Custom Properties. Not only do these properties ensure that each element is delivered according to your application’s specifications, but for all color/scale/content direction specifications across Spectrum.
```
Spectrum prefers Spanish to British, we should, too.

File: documentation/guides/getting-started.md [hunk 4]
```diff
@@ -23,13 +27,10 @@ The code above renders to the browser as follows (be patient while the JS is loa
...
+When you're ready to look into more advanced usage of the components and themes in your application, there are vanilla CSS implementations of these tokens available in the `@spectrum-web-components/styles` package. However, to get started immediately, the `<sp-theme>` element delivers these tokens to a scoped HTML context and is what’s used in the code above. You'll notice the usage of `scale="large"` and `color="dark"` to outline the scale and color applied to this theme context, respectively. To make a theme context with `scale="medium"` and `color="lightest"`, you can use the following code sample:
```
Comment (Westbrook) [inline documentation/guides/getting-started.md:33]:
```suggestion
When you're ready to look into more advanced usage of the components and themes in your application, there are vanilla CSS implementations of these tokens available in the `@spectrum-web-components/styles` package. However, to get started immediately, the `<sp-theme>` element delivers these tokens to a scoped HTML context and is what is used in the code above. You'll notice the usage of `scale="large"` and `color="dark"` to outline the scale and color applied to this theme context, respectively. To make a theme context with `scale="medium"` and `color="lightest"`, you can use the following code sample:
```

File: documentation/guides/getting-started.md [hunk 5]
```diff
@@ -47,6 +48,6 @@ Visually, all Spectrum Web Components are an expression of the design tokens tha
 
 ## What you can do
 
-From this baseline, visit the documentation for each individual component package and find one, some, or all of those that are right for your project. Start building today and use rapid, component-based prototyping to bring your designs to life faster than ever.
+Now that you have a starting point, visit the documentation for each package and find some components that are right for your project. You can take a look at our examples on webcomponents.dev to get an idea of how these components behave in a development environment, or browse through our <sp-link href="https://opensource.adobe.com/spectrum-web-components/storybook/" target="_blank">storybook</sp-link>. Once you start developing, you’ll find that rapid, component-based prototyping brings your designs to life faster than ever.
```
Comment (Westbrook) [inline documentation/guides/getting-started.md:51]: Can you link `webcomponents.dev` to https://webcomponents.dev/workspace/adobe?collection=spectrum-web-components

File: documentation/guides/getting-started.md [hunk 6]
```diff
@@ -47,6 +48,6 @@ Visually, all Spectrum Web Components are an expression of the design tokens tha
...
+When you find a component or two you’d like to use, take a look at our <sp-link href="https://github.com/adobe/spectrum-web-components/tree/main/projects/example-project-rollup" target="_blank">Rollup</sp-link> and/or <sp-link href="https://github.com/adobe/spectrum-web-components/tree/main/projects/example-project-webpack" target="_blank">Webpack</sp-link> example projects for recommendations on how to start preparing your project for production. Leverage the listings under the “Usage” section on each component’s documentation page to support the patterns you'll find therein. And then, when you're ready... ship, ship, ship! We look forward to seeing what you bring to life.
```
Comment (Westbrook) [inline documentation/guides/getting-started.md:53]:
```suggestion
When your ready to deploy your app to production, take a look at our <sp-link href="https://github.com/adobe/spectrum-web-components/tree/main/projects/example-project-rollup" target="_blank">Rollup</sp-link> and/or <sp-link href="https://github.com/adobe/spectrum-web-components/tree/main/projects/example-project-webpack" target="_blank">Webpack</sp-link> example projects for recommendations on how to optimize your project for that environment. Leverage the listings under the “Usage” section on each component’s documentation page to support the patterns you'll find therein. And then, when you're ready... ship, ship, ship! We look forward to seeing what you bring to life.
```
Maybe...

**Review by Westbrook — COMMENTED** [review]
This is great!

Nits...

File: documentation/guides/getting-started.md (hunk 1 above)
Comment (najikahalsema) [reply documentation/guides/getting-started.md:7]: I think a comma here is fine.

File: documentation/guides/getting-started.md (hunk 2 above)
Comment (najikahalsema) [reply documentation/guides/getting-started.md:21]: LOL, that's right...

File: documentation/guides/getting-started.md (hunk 5 above)
Comment (najikahalsema) [reply documentation/guides/getting-started.md:51]: Whoops, forgot about this. Added

**Review by Westbrook — APPROVED** [review]
LGTM!

**Pull request merged**